lib/routes/papers/index.ts

```diff
@@ -81,19 +92,25 @@ export const route: Route = {
     url: 'papers.cool',
     maintainers: ['nczitzk', 'Muyun99'],
     handler,
-    example: '/papers/arxiv/cs.AI',
-    parameters: { category: 'Category, arXiv Artificial Intelligence (cs.AI) by default' },
+    example: '/papers/arxiv/cs.AI or /papers/query/Detection',
```

Comment:
This example will be converted into `"https://rsshub.app/papers/arxiv/cs.AI or /papers/query/Detection"`